app/src/main/java/com/example/mykotlincalculator21/utils/ErrorUtils.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.mykotlincalculator21.utils
+
+enum class ErrorUtils {
```

Comment:
rename de ErrorUtils to ResultUtils as we talked

Reply:
Done. thx